src/Authentication/ClientCertificate/ClientCertificateClient.php

```diff
@@ -21,12 +21,18 @@ public function __construct(\Phpro\SoapClient\Caller\Caller $caller)
     }
 
     /**
-     * @param RequestInterface|Type\GetServiceTicket $getServiceTicketPart
-     * @return ResultInterface|Type\GetServiceTicketResponse
+     * @param RequestInterface & Type\GetServiceTicket $getServiceTicketPart
+     * @return ResultInterface & Type\GetServiceTicketResponse
```

Comment:
think we should indicate one or the other, usually its the interface, but if we need it to be the class then lets go with that

Reply:
This code is generated by the library, check the readme [here](https://github.com/openeuropa/epoetry-client/blob/2.x/README.md#generate-soap-related-classes).